pkg/util/kubernetes/kubelet/containers.go

```diff
@@ -3,20 +3,21 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2018 Datadog, Inc.
 
-// +build kubelet,linux
```

Comment:
Code in this file calls `container.SetCgroups` and friends which are linux only, hence the additional `linux` build tag

Reply:
My bad this was for testing purposes, reverting